example_versioned_docs/version-latest/23-Json.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Json
+id: json
+slug: /json
+sidebar_label: 23. Json
+description: Create Json values in Wing
+keywords: [Wing language, json]
+---
+
+```js playground example title="main.w"
```

Comment:
````suggestion
---

Wing has a dedicated type named `Json` for representing [JSON](https://www.json.org/json-en.html). A `Json` value can be an object, but it can also be an array, string, boolean, number, or null.

```js playground example title="main.w"
````

Reply:
BTW, there's actually no way to create a JSON null value or check for it in Wing explicitly (see https://github.com/winglang/wing/issues/1819) - this is something that we should probably address before 1.0